Makefile

```
@@ -87,6 +87,10 @@ lint: golangci-lint yamllint ## Run golangci-lint linter & yamllint
 lint-fix: golangci-lint ## Run golangci-lint linter and perform fixes
 	$(GOLANGCI_LINT) run --fix
 
+.PHONY: lint-test
+lint-test: golangci-lint ## Run golangci-lint for scaffolded project
+	./lint-testdata.sh
```

```suggestion
.PHONY: lint-testada
lint-testada: golangci-lint ## Run golangci-lint for the lscaffolded projects under testdata directory
	./lint-testada
```
Could we call something like lint-testdata or lint-samples to be more intuitive